.gitignore

```diff
@@ -1 +1,5 @@
+# Cache
 __pycache__
+
+# Editor folders
```

Comment:
@Dekat to me those folders do not belong this repository.
It is expected from developers to use a global system-wide `.gitignore` for files that are related to the tools they use for development.

Reply:
:scream: We can do that?!

Reply:
@quentinus95 Fixed here: https://github.com/MeilleursAgents/backend-technical-test/pull/15/commits/8280fe717ce0b73dc21224e5b5e23bacea79632f
And thanks again for this great tip!